marketplace/wpp_templates/models.py

```diff
@@ -0,0 +1,116 @@
+import uuid
+
+from django.db import models
+
+from rest_framework.exceptions import ValidationError
+
+
+class TemplateMessage(models.Model):
```

Comment:
This model relates to the App
![image](https://user-images.githubusercontent.com/49874732/184200136-e6a20894-1a8d-4719-a701-1da47c4e3889.png)